Source/Common/RootFSSetup.cpp

```diff
@@ -221,13 +221,14 @@ bool Setup(char **const envp) {
   // If it is setup to use a squashfs then we need to do something more complex
 
   FEX_CONFIG_OPT(LDPath, ROOTFS);
-  auto ContainerPrefix = FEXCore::Config::FindContainerPrefix();
-  if (!ContainerPrefix.empty()) {
-    // If we are inside of a rootfs/container then drop the rootfs path
-    // Root is already our rootfs
-    FEXCore::Config::Erase(FEXCore::Config::CONFIG_ROOTFS);
-    return true;
-  }
+  // XXX: Disabled for now. Causing problems due to weird filesystem problems
+  //auto ContainerPrefix = FEXCore::Config::FindContainerPrefix();
```

Comment:
How about using `if (false)` instead of a block comment so it's ensured this keeps compiling properly in the future?

Is there any way to provide more detail about the issue (what goes wrong? how to reproduce?) for future reference?

Reply:
Switched over to an if statement and added more to the comment about how to reproduce.